### DISCUSSION TITLE
had a query on using this action on a job which checkouts multiple projects

### DISCUSSION THREAD

**Discussion by suslovjb** [wzieba/Firebase-Distribution-Github-Action#55] [Q&A] [answered]
i have an android and react native repos which i will checkout in a single job. 

when i use firebase distribution github action, it will not be run in android repo folder.
but i have provided the apk file path properly

 - name: Upload artifact to Firebase App Distribution
        uses: wzieba/Firebase-Distribution-Github-Action@v1
        with:
          appId: ${{secrets.FIREBASE_APP_ID}}
          token: ${{secrets.FIREBASE_TOKEN}}
          groups: qa
          file: androidproject/app/build/outputs/apk/sprint/app-sprint.apk

i am getting an error like the below

Error: failed to upload release. HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.

i have used firebase login:ci to generate token and add it to secrets

token was generated like below

Waiting for authentication...

✔  Success! Use this token to login on a CI server:

1//1ffsfs33fssf1131....sfffsf12fsfs

Any input on this?

**Comment by mateuszkwiecinski** [comment, marked as answer]
you can try setting `debug: true` and see if that reveals any additional information. 
And just shooting in the dark I'd verify if `FIREBASE_APP_ID` secret is properly set (i.e. if there isn't a typo somewhere). The config you pasted looks fine and should be working 🤷‍♂️

**Comment by wzieba** [comment]
To close the loop: the action is working correctly (see passed test scenario [here](https://github.com/wzieba/Firebase-Distribution-Github-Action/runs/5806759755?check_suite_focus=true) ). It must be something wrong with your config - please see https://github.com/wzieba/Firebase-Distribution-Github-Action/issues/10 for detailed instructions.
